pkg/component/crddeployer/crd_test.go

```diff
@@ -32,6 +35,26 @@ var _ = Describe("CRD", func() {
 		applier    kubernetes.Applier
 		testClient client.Client
 
+		removeFinalizerOnDeletionAnnotation = func(ctx context.Context, crdName string, c client.Client) {
+			Expect(retry.Until(ctx, 1*time.Second, func(ctx context.Context) (done bool, err error) {
```

Comment:
To ensure the right calling line to be reported in case of an assertion failure
```suggestion
			ExpectWithOffset(1, retry.Until(ctx, 1*time.Second, func(ctx context.Context) (done bool, err error) {
```